lib/toxiproxy.rb

```diff
@@ -199,8 +203,12 @@ def toxics(direction)
 
   private
 
+  def self.uri
+    @uri ||= ::URI.parse(ENV.fetch('TOXIPROXY_HOST', 'http://127.0.0.1:8474'))
```

Comment:
Agreed, in any language. I was caught up in a WIP thing that I'm working on where I've been poking at things with env vars.